src/styles/utilits/_mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin position {
+  bottom: 50%;
+  left: 50%;
+  transform-origin: bottom center;
+}
```

Comment:
need to add selfposition 😅 
<img width="462" alt="image" src="https://user-images.githubusercontent.com/99652389/180778370-59c8d9cd-a52b-4aff-850c-21b793b3559e.png">

Reply:
but  don't use margins)

Reply:
you mean that the arrow was in the center of the circle?

Reply:
Sorry for slowpoky answers! You guys finished your HWs faster than I can check them 😄